coss/base/templates/base.html

```diff
@@ -115,7 +115,6 @@
       <script type="text/javascript" src="{% static 'lib/jquery/dist/jquery.min.js' %}"></script>
       <script type="text/javascript" src="{% static 'lib/popper.js/dist/umd/popper.min.js' %}"></script>
       <script type="text/javascript" src="{% static 'lib/bootstrap/dist/js/bootstrap.min.js' %}"></script>
-      <script type="text/javascript" src="{% static 'js/navbar.js' %}"></script>
```

Comment:
Can you please add a short summary on why we are dropping the `navbar`?

Reply:
This js code was used to switch the logo when you scroll down, but the logo we used there was out of brand guidelines. So we'll just stick with the same logo we regularly use.